cartographer_ros/cartographer_ros/map_builder_bridge.h

```diff
@@ -44,32 +44,33 @@ class MapBuilderBridge {
     TrajectoryOptions trajectory_options;
   };
 
-  MapBuilderBridge(const NodeOptions& node_options, tf2_ros::Buffer* tf_buffer);
+  MapBuilderBridge(const NodeOptions &node_options, tf2_ros::Buffer *tf_buffer);
```

Comment:
Ah, I see. But it's odd that it decided to this - this style was never the majority.